lib/mindwendel_web/components/core_components.ex

```diff
@@ -380,8 +384,8 @@ defmodule MindwendelWeb.CoreComponents do
         name={@name}
         class={[
           "form-control",
-          @errors == [] && "is-valid",
-          @errors != [] && "is-invalid"
+          @used? && @errors == [] && "is-valid",
+          @used? && @errors != [] && "is-invalid"
```

Comment:
also, this is repeated three times and will change all the time - but it still seemed too small/odd for a helper. Although, now that I think about it there could be a `validation_class` helper as also there should never be both of them :thinking: